Kanchan-Wakchaure/SER517-Group2-Fall2019
# Us 26 task 27 - Improved preview route with road directions

To test:
1. Create events and go to preview route age. 
2. Car should move towards events in correct order on the road.
3. Car direction changes accordingly.

Note: Home address is still hard-coded.
